lab_1_keywords_tfidf/main.py

```diff
@@ -66,10 +88,20 @@ def get_top_n(frequencies: dict[str, Union[int, float]], top: int) -> Optional[l
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if isinstance(frequencies, dict) and isinstance(top, int):
+        top_dic = {}
+        top_values = sorted(frequencies.values(), reverse=True)
```

Comment:
if there a way to sort the keys of dictionary by its values? 
inspect the `sorted` function documentation for useful arguments